test/bindings-test.js

```diff
@@ -11,12 +11,13 @@ var canBatch = require('can-event/batch/batch');
 var viewCallbacks = require('can-view-callbacks');
 var canCompute = require('can-compute');
 var canViewModel = require('can-view-model');
-require('can-util/dom/events/inserted/');
 
 var stacheExpression = require('can-stache/src/expression');
 
 var domData = require('can-util/dom/data/data');
 var domMutate = require('can-util/dom/mutate/mutate');
+var domEvents = require('can-util/dom/events/');
+require('can-util/dom/events/inserted/');
```

Comment:
this needs to be `inserted/inserted` and `events/events` for it to work w/ all the builds (and if we ever get this directly loadable by node).